packages/manager/modules/core/src/manager-core.constants.js

```diff
@@ -59,6 +59,28 @@ export const URLS = {
     SN: 'https://www.ovh.sn/support/',
     TN: 'https://www.ovh.com/tn/support/',
   },
+  support_level: {
+    ASIA: 'https://www.ovhcloud.com/asia/support-levels',
+    AU: 'https://www.ovhcloud.com/en-au/support-levels',
+    CA: 'https://www.ovhcloud.com/en-ca/support-levels',
+    DE: 'https://www.ovhcloud.com/de/support-levels',
+    ES: 'https://www.ovhcloud.com/es-es/support-levels',
+    FR: 'https://www.ovhcloud.com/fr/support-levels',
+    GB: 'https://www.ovhcloud.com/en-gb/support-levels',
+    IE: 'https://www.ovhcloud.com/en-ie/support-levels',
+    IT: 'https://www.ovhcloud.com/it/support-levels',
+    MA: 'https://www.ovhcloud.com/fr-ma/support-levels',
+    NL: 'https://www.ovhcloud.com/nl/support-levels',
+    PL: 'https://www.ovhcloud.com/pl/support-levels',
+    PT: 'https://www.ovhcloud.com/pt/support-levels',
+    QC: 'https://www.ovhcloud.com/fr-ca/support-levels',
+    SG: 'https://www.ovhcloud.com/en-sg/support-levels',
+    SN: 'https://www.ovhcloud.com/fr-sn/support-levels',
+    TN: 'https://www.ovhcloud.com/fr-tn/support-leve',
+    US: 'https://www.ovhcloud.com/en/support-levels',
+    WE: 'https://www.ovhcloud.com/en/support-levels',
+    WS: 'https://www.ovhcloud.com/es/support-levels',
```

Comment:
no default URL? may GB can be considered as a default one?

Reply:
Good point, added. Thanks